tests/integration/internal/test_cache.py

```diff
@@ -0,0 +1,55 @@
+from spccore.internal.cache import *
+
+
+def test_setup(cache, file_handle_id, file_path):
+    exist_file_paths = cache.get_all_unmodified_cached_file_paths(file_handle_id)
+
+    file_path = normalize_path(file_path)
+    assert file_path not in exist_file_paths
+
+
+def test_register_example(cache, file_handle_id, file_path):
+    cache.register(file_handle_id, file_path)
+
+    file_path = normalize_path(file_path)
+    exist_file_paths = cache.get_all_unmodified_cached_file_paths(file_handle_id)
+    assert len(exist_file_paths) == 1
+    assert file_path in exist_file_paths
+
+
+def test_download_default_location_empty_cache(cache, file_handle_id, file_path):
```

Comment:
how is this test different from test_register_example? the only parameter that changed is the file path given to cache.register

Reply:
uhm, I was experiment different cases of download:
1. download to user defined path (test_register_example)
2. download to default cache location (this test)
You are right. This was a proof of concept that I can write download function with these expose cache's functions. However, it's redundant. Will remove the test_register_example one.